libs/ui/src/components/layouts/SplitColumn/SplitColumn.tsx

```diff
@@ -0,0 +1,32 @@
+import { StyledRow, StyledSplitColumn } from './SplitColumn.styles';
+
+// REVIEW: No stories until there's a solid form sub-layout
+
+type SplitColumnProps = {
+  singleRow?: Row;
+  rows?: Row[];
+};
+export const SplitColumn = ({ rows, singleRow }: SplitColumnProps) => {
+  return (
+    <StyledSplitColumn>
+      {singleRow && <Row {...singleRow} />}
+      {rows?.map((row) => {
+        return <Row key={row.rowID} {...row} />;
+      })}
```

Comment:
Why a prop for a single row and another for an array of rows?
Maybe something to check if the prop is an array.
```
type SplitColumnProps = {
  row: Row | Row[];
};

    if (Array.isArray(rows)) {
      ...
    }
    or 
    if (props.data instanceof Array){
      ...
    }
```